en/docs/concepts/intro-authentication.md

```diff
@@ -1,19 +1,22 @@
 # Introduction to Authentication
 
 ### User Identity
-The way in which a user can be uniquely identified is called ‘User Identity’. A user’s identity can differ on the 
-platform and the use case. Considering the digital world a single user can have different identities as user name, user 
-id, biometric information, etc.
+The way in which a user can be uniquely identified is called ‘User
+Identity’. A user’s identity can differ on the platform and the use
+case. In the digital world a single user can have different identities
+as user name, user id, email id, biometric information and etc.
```

Comment:
The way in which a user **is** uniquely identified is called **a** ‘User
Identity’. A user’s identity can differ **based** on the platform and the use
case. In the digital world{insert comma} a single user can have different **identifying attributes**
**such** as user name, user **ID**, email **ID**, biometric information**,** etc.

Reply:
@DMHP, please node